src/cli/templates/create/_._editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig: http://EditorConfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{package.json,package-lock.json,*.yml}]
```

Comment:
Can be dropped, same rules as generic

Reply:
Yes, unless they change the generic one to tabs or 4 spaces. Then `npm install --save` will reformat `package.json` with 2 spaces, and their IDE to 4 spaces :)

Reply:
Fair enough